src/geodesic.rs

```diff
@@ -842,7 +849,7 @@ impl Geodesic {
             }
 
             let alp12: f64;
-            if !meridian && comg12 > -0.7071 && sbet2 - sbet1 < 1.75 {
+            if !meridian && comg12 > -FRAC_1_SQRT_2 && sbet2 - sbet1 < 1.75 {
```

Comment:
I understand why you're doing this, but it diverges from our reference implementation whose test suite we validate against.

The tests seem to all still be passing, which makes me wonder if this edge case is maybe not tested too precisely. It could be a real pain to track down later if the tests input changes to test around this more precisely.

Still, your proposed change *feels* like the right thing to do. Could you add a comment to the effect of: 
> `we're diverging from Karney's implementation here which uses the hardcoded constant: -0.7071 for FRAC_1_SQRT_2`

Reply:
I had the same doubts seeing this suggestion from Clippy. But I decided that it is beneficial to do that because use of internal constant improves code readability and has an explicit type.

This edge case might indeed not be tested precisely enough. Actually, probability for hitting a different branch in Rust implementation than in original implementation is _very_ low: difference between constants is roughly `-6.78e-6` and there are two other conditions alongside.

And the constant `-0.7071` is present only in Karney's implementation but not in the reference paper. So it seems like its use is an implementation shortcut. 

I've added the requested change.